src/forwarders.js

```diff
@@ -387,23 +387,30 @@ export default function Forwarders(mpInstance, kitBlocker) {
         }
     };
 
-    this.callSetUserAttributeOnForwarders = function(key, value) {
+    this.onHandleForwarderUserAttributes = function(key, value, functionName) {
         if (kitBlocker && kitBlocker.isAttributeKeyBlocked(key)) {
             return;
         }
 
         if (mpInstance._Store.activeForwarders.length) {
             mpInstance._Store.activeForwarders.forEach(function(forwarder) {
+                var forwarderFunction = forwarder[functionName];
                 if (
-                    forwarder.setUserAttribute &&
+                    forwarderFunction &&
                     forwarder.userAttributeFilters &&
                     !mpInstance._Helpers.inArray(
                         forwarder.userAttributeFilters,
                         mpInstance._Helpers.generateHash(key)
                     )
                 ) {
                     try {
-                        var result = forwarder.setUserAttribute(key, value);
+                        var result;
+
+                        if (functionName === 'setUserAttribute') {
```

Comment:
`setUserAttribute` and `removeUserAttribute` should be made into constants so we can know if we made a typo.